### ISSUE TITLE
Can not reproduce the performance of self-supervised learning model (Moco)

### ISSUE THREAD

**Issue by ZMHH-H** [facebookresearch/SlowFast#554]
Hi, thanks for your excellent work and sharing the paper code about self-supervised learning!
I'm tring to reproduce the result of Moco method on UCF101 split1 dataset. So I download the Moco pretrained model (ρ=2) in the Readme file and get this checkpoint: MoCo_SlowR50_8x8_T2_epoch_00200.pyth
Then I finetuned the model on UCF101 trainsplit1 with configs: [finetune_ucf_Slow_R50_syn8.yaml]. The only difference is my batchsize is 16 for 2 GPUS, so i set the base learning rate to one fourth of the original.(The original lr is 0.01 for 64 batch size, mine is 0.0025).
Here is my config:
![2022-06-07 16-29-28 的屏幕截图](https://user-images.githubusercontent.com/48144545/172334259-eb2e92c7-d9d3-4c8b-9913-85117803e86c.png)
But finally i only get 69.39% top1 Acc on UCF101 testsplit1. Could you please figure out what i did wrong?
